test/e2e_test.ts

```diff
@@ -11,15 +11,16 @@ import * as closure from 'google-closure-compiler';
 
 import {goldenTests} from './test_support';
 
-export function checkClosureCompile(
-    jsFiles: string[], externsFiles: string[], done: (err?: Error) => void) {
+export function checkClosureCompile(jsFiles: string[], externsFiles: string[], done: DoneFn) {
   const startTime = Date.now();
   const total = jsFiles.length;
   if (!total) throw new Error('No JS files in ' + JSON.stringify(jsFiles));
 
   const CLOSURE_COMPILER_OPTS: closure.CompileOptions = {
     'checks_only': true,
     'jscomp_error': 'checkTypes',
+    // TODO(martinprobst): enable when NTI is ready. Currently doesn't support enum union types.
```

Comment:
It looks like you worked around the enum thing elsewhere, so adjust this comment.  I kinda suspect we should just avoid NTI until a user demands it.

Reply:
Whoops, leftover comment. I changed only the comment, I think it's useful for us to run in NTI, just for the more precise error checks.